eng/pipelines/templates/stages/cosmos-sdk-client.yml

```diff
@@ -143,12 +145,12 @@ stages:
 
   # The Prerelease and Release stages are conditioned on whether we are building a pull request and the branch.
   - ${{if and(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['System.TeamProject'], 'internal'))}}:
-    - template: archetype-java-release.yml
+      - template: archetype-java-release.yml
```

Comment:
This indention looks incorrect.

Reply:
Definitely don't understand how this snuck in.

Reply:
Thanks for the catch. I'm going to update this.